model/lib/status-log.js

```diff
@@ -13,6 +13,15 @@ module.exports = memoize(function (db) {
 		time: { type: db.DateTime, required: true },
 		official: { type: User },
 		officialFullName: { type: StringLine },
-		text: { type: db.String, required: true }
+		text: { type: db.String, required: true },
+		toJSON: { value: function (ignore) {
+			return {
+				label: this.getOwnDescriptor('label').valueToJSON(),
+				time: this.getOwnDescriptor('time').valueToJSON(),
+				official: this.getOwnDescriptor('official').valueToJSON(),
+				officialFullName: this.getOwnDescriptor('officialFullName').valueToJSON(),
```

Comment:
We should add it only if we have `officialFullName` (e.g. it doesn't apply to requirement uploads now).
Otherwise we create records as `officialFullName: undefined` which bring noise to snapshot and unjustifiably increase their size.